### ISSUE TITLE
SVG elements with the `svg:` tag prefix are stripped.

### ISSUE THREAD

**Issue by adroitwhiz** [cure53/DOMPurify#492]
### Background & Context

https://github.com/llk/scratch-svg-renderer recently began to use DOMPurify on all SVGs it renders. I noticed that certain SVGs stopped rendering after it did so.

### Bug

Certain SVGs from Inkscape (I have attached an example) are stripped of almost all tags. It seems this is because the tags in said SVGs are prefixed with `svg:`, and such prefixed tags are stripped.

#### Input

`<svg:svg xmlns:svg="http://www.w3.org/2000/svg" xmlns="http://www.w3.org/2000/svg"></svg:svg>`

A full example: [incorrectlysanitized.svg.txt](https://github.com/cure53/DOMPurify/files/5611319/incorrectlysanitized.svg.txt) (change the extension to .svg)

#### Given output

(empty string)

#### Expected output

`<svg xmlns="http://www.w3.org/2000/svg"></svg>` (I think)

**Comment by cure53** [comment]
Heya, it should be easy to address that using a config option:

```javascript
const xss = `<svg:svg xmlns:svg="http://www.w3.org/2000/svg" xmlns="http://www.w3.org/2000/svg"></svg:svg>`;
const html = DOMPurify.sanitize(xss, {ADD_TAGS: ['svg:svg']});
console.log(html)
```

Or with a hook, if you feel adventurous:

```javascript
DOMPurify.addHook('uponSanitizeElement', function(node){
  try {
      node.innerHTML = node.innerHTML.replace(/<svg:/ig, '<');
  } catch(e) {}
});
const xss = `<svg:svg xmlns:svg="http://www.w3.org/2000/svg" xmlns="http://www.w3.org/2000/svg"></svg:svg>`;
const html = DOMPurify.sanitize(xss);
console.log(html)
```

**Comment by adroitwhiz** [comment]
I think the title may have been a bit ambiguous-- this happens for all elements in the SVG spec that are prefixed with `svg:`, so e.g. `<svg:g>`, `<svg:path>`, etc.

**Comment by cure53** [comment]
Yup, still something that is best addressed using config options or hooks. 

I see no need for a change to the core library here. If it can be done via config, it shall be done via config :)

**Comment by cure53** [comment]
Closing this for now, can be fixed via config or hook.
